tests/test_cli.py

```diff
@@ -0,0 +1,80 @@
+import pytest
+from typer.testing import CliRunner
+
+from gtfs.__main__ import app
+from gtfs.utils.constants import *
+from gtfs.utils.geom import bbox_contains_bbox, bbox_intersects_bbox
+
+
+@pytest.fixture(scope="module")
+def runner():
+    return CliRunner()
+
+
+class Test_Geom_Functions:
```

Comment:
this is not testing cli stuff, but the `util.geom` module. better to create a new file `test_util_geom.py` or `tests/test_utils/test_geom.py`.